cmd/cmd.go

```diff
@@ -373,6 +375,40 @@ func CmdStart(c *cli.Context, cfg *config.Config, endpointServe func(cfg *config
 	return nil
 }
 
+func CmdImportEthAccount(c *cli.Context) error {
+	keyfile := c.Args().First()
+	if len(keyfile) == 0 {
+		return fmt.Errorf("keyfile must be given as argument")
+	}
+	key, err := crypto.LoadECDSA(keyfile)
+	if err != nil {
+		return err
+	}
+
+	var cfg struct {
+		KeyStore config.ConfigKeyStore `validate:"required"`
+	}
+	if err := config.LoadFromCliFlag(c, &cfg); err != nil {
+		return err
+	}
+
+	ks := keystore.NewKeyStore(cfg.KeyStore.Path, keystore.StandardScryptN, keystore.StandardScryptP)
+
+	account, err := ks.ImportECDSA(key, cfg.KeyStore.Password)
+	if err != nil {
+		return err
+	}
+
+	log.WithFields(log.Fields{
+		"keyfile": keyfile,
+		"address": account.Address.Hex(),
+	}).Info("Imported Eth Account from private key")
+
+	printAccountToml(account)
```

Comment:
`printAccountToml` has a hardcoded message that says "Ethereum Account created successfully." But in this case it's not created but imported.  Maybe the message should be outside of `printAccountToml`

Reply:
Done